packages/ui/src/components/CategoryBlock/CategoryBlock.tsx

```diff
@@ -0,0 +1,40 @@
+import { ArrowRightIcon } from '@heroicons/react/24/outline';
+import type { Category, Culture } from '@prezly/sdk';
+import { getCategoryUrl, getLocalizedCategoryData, LocaleObject } from '@prezly/theme-kit-core';
+import { twMerge } from 'tailwind-merge';
+
+import { Link } from '../Link';
+
+export interface Props {
+    category: Category;
+    className?: string;
+    locale: Culture['code'];
+}
+
+export function CategoryBlock({ className, category, locale }: Props) {
+    const localeObj = LocaleObject.fromAnyCode(locale);
+    const { name, description } = getLocalizedCategoryData(category, localeObj);
+
+    return (
+        <div
+            className={twMerge(
+                'flex flex-col p-6 w-full md:w-64 rounded border border-gray-200',
+                className,
+            )}
+        >
+            <p className="subtitle-small">{name}</p>
```

Comment:
IMO we should use heading tags for card titles to improve semantics. An `h3` or `h4` would work well here.